xla/service/gpu/gemm_algorithm_picker.cc

```diff
@@ -233,13 +232,13 @@ StatusOr<AutotuneResult> DoGemmAutotuneNoCache(
   VLOG(3) << "Starting autotune of GemmThunk " << gemm->ToString();
   se::DeviceMemoryAllocator* allocator = autotune_config.GetAllocator();
   TF_ASSIGN_OR_RETURN(se::Stream* const stream, autotune_config.GetStream());
-  GemmBackendConfig gemm_config =
+  GemmBackendConfig backend_config =
```

Comment:
It would be better to have simple renames in a separate CL.

Reply:
ok, sure, I will place it in a separate PR then. My point is that the naming convention is quite unfortunate here: there are GemmBackendConfig gemm_config and GemmConfig config variables used the same function..